### PR TITLE
Preferred node in price-preferred expander

### PR DESCRIPTION
**Pull request by mwielgus**
Ref: #82

### CODE REVIEW

File: cluster-autoscaler/expander/price/preferred.go [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+	nodeLister kube_util.NodeLister
+}
+
+func (spnp *simplePreferredNodeProvider) Node() (*apiv1.Node, error) {
+	nodes, err := spnp.nodeLister.List()
+	if err != nil {
+		return nil, err
+	}
+	size := len(nodes)
+	glog.Warningf("Nodes = %d", size)
```
Comment (MaciekPytel) [inline cluster-autoscaler/expander/price/preferred.go]: That looks debug-y and probably shouldn't be here

File: cluster-autoscaler/expander/price/preferred.go [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+	mb := int64(1024 * 1024)
+	cpu := int64(1000)
+
+	// Double node size with every time the cluster size increases 3x.
+	if size <= 2 {
+		return buildNode(1*cpu, 3750*mb), nil
```
Comment (MaciekPytel) [inline cluster-autoscaler/expander/price/preferred.go]: I wondered if this shouldn't be somehow connected to cloudprovider to prefer machine sizes available for that specific provider?

Comment (MaciekPytel) [inline cluster-autoscaler/expander/price/preferred.go]: To clarify: I'm not saying it should be connected, just giving an idea.

File: cluster-autoscaler/expander/price/preferred_test.go [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import (
+	"testing"
+
+	. "k8s.io/autoscaler/cluster-autoscaler/utils/test"
+	apiv1 "k8s.io/kubernetes/pkg/api/v1"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type nodeLister struct {
```
Comment (MaciekPytel) [inline cluster-autoscaler/expander/price/preferred_test.go]: I would prefer to include the word 'test' in the name of this struct

Comment (MaciekPytel) [inline cluster-autoscaler/expander/price/preferred_test.go]: I would prefer to include the word 'test' in the name of this struct

File: cluster-autoscaler/expander/price/preferred.go (hunk 2 above)
Comment (mwielgus) [inline cluster-autoscaler/expander/price/preferred.go]: Maybe in some later version... It doesn't matter much whether this is a machine from cloud provider or not. Overall shape/size is important.

File: cluster-autoscaler/expander/price/preferred_test.go (hunk 3 above)
Comment (mwielgus) [inline cluster-autoscaler/expander/price/preferred_test.go]: Done.

Comment (mwielgus) [inline cluster-autoscaler/expander/price/preferred_test.go]: Done.

**Pull request merged**